### PR TITLE
Added table footer

### PR DESCRIPTION
**Pull request by lucascumsille**
This contains the style for sharing the table's info in social media.
Screenshot 2021-12-24 at 10 48 38

### CODE REVIEW

**Review by zarino — APPROVED** [review]
Two small stylistic comments inline. Take them or leave them.

Also, I have a feeling the SVGs could be compressed quite a bit with a run through `svgo` or similar.

File: scoring/static/scss/tables.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    width: 32px;
+    height: 32px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    @include transition(background-image 0.3s ease-in-out);
+}
+
+.table-social-icon.facebook {
+    background-image: url('../img/icon-facebook.svg');
+    &:hover {
```
[the remark this answers is not in the document]
Comment (lucascumsille) [reply scoring/static/scss/tables.scss]: Thanks for the feedback. I just pushed the all those fixes, plus some extra one related with this feedback.

[the remark this answers is not in the document]
Comment (lucascumsille) [reply scoring/static/scss/tables.scss]: @zarino btw none of those changes are urgent, no need to review today.

**Review by zarino — APPROVED** [review]
:+1:

**Pull request merged**